src/hooks/use-row-selection/use-row-selection.spec.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { render } from '../../test-utils';
+import useRowSelection from './use-row-selection';
+
+const testData = [
+  { id: 'system-crasher' },
+  { id: 'birds-of-passage' },
+  { id: 'woman-at-war' },
+];
+
+const TestComponent = props => {
+  const {
+    rows,
+    toggleRow,
+    selectRow,
+    deselectRow,
+    selectAllRows,
+    deselectAllRows,
+    getIsRowSelected,
+    getNumberOfSelectedRows,
+    // eslint-disable-next-line react/prop-types
```

Comment:
On this line? Should we disable it for the file or specs in general?

Reply:
It just complains about the props on the next line. But this is just a test component so idc about adding extra boilerplate.

I'll move to the top of the file though.